### PR TITLE
Feature webinterface entities

### LINKED ISSUE #138: Web Interface - Display Entities
Entities must be displayed on the map of the web interface.

This task is the implementation of this feature.

This task is a subtask of #43 Web Interface.

### PR DESCRIPTION
**Pull request by Geweldig**
Entities are now rendered on the map.
There are no PMD/Checkstyle/findbugs warnings.
All new java methods have been tested.

Closes #138

### CODE REVIEW

File: src/main/java/nl/tudelft/contextproject/model/Entity.java [hunk 1]
```diff
@@ -46,4 +46,14 @@ public boolean collidesWithPlayer(float dist) {
 		Vector3f thisLoc = getSpatial().getLocalTranslation();
 		return thisLoc.distance(playerLoc) < dist;
 	}
+
+	/**
+	 * Get the location of an entity.
+	 * @return
+	 * 			a Vector3f representing the location
+     */
```
Comment (Taeir) [inline src/main/java/nl/tudelft/contextproject/model/Entity.java]: :O Incorrect indentation.

Comment (BCrlrd) [inline src/main/java/nl/tudelft/contextproject/model/Entity.java]: Which only appears on github.

**Comment by Taeir** [comment]
Looks very good, just one little indentation mistake :P.

**Comment by ChielBruin** [comment]
Looks good! I will merge it

**Pull request merged**